internal/worker/chain.go

```diff
@@ -351,64 +292,40 @@ func (c *Chain) process(ctx context.Context, startBlockNumber, endBlockNumber in
 			AttestationIdBytes: unpackedData.AttestationId,
 			AttestationIdStr:   hex.EncodeToString(unpackedData.AttestationId[:]),
 			Attestation:        unpackedData.Attestation,
-		})
+		}
 
-		attestationIds = append(attestationIds, unpackedData.AttestationId)
 		// Verify attestation
-		isTrue, err := verifyAttestation(c, unpackedData.Attestation)
-		if !isTrue {
-			result = append(result, false)
-			continue
+		result, err := verifyAttestation(c, unpackedData.Attestation)
+		if err != nil {
+			return errors.Wrap(err, "verify attestation error")
```

Comment:
Using `return` will cause the program to stop at the current block, and other transaction logs cannot be processed. It is recommended to use `continue` to skip the current erroneous attestation.

Reply:
Fixed!